docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -9,8 +9,7 @@ times, select the *Rule Monitoring* tab on the *Rules* page (*Detect* ->
 [role="screenshot"]
 image::images/monitor-table.png[]
 
-For detailed information on a rule, its generated alerts, and its errors, click on
-a rule name in the *All rules* table.
+For detailed information on a rule, the alerts it generated, and associated errors, click on a rule's name in the *All rules* table. This also allows you to perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.
```

Comment:
To me, it looks like, `This allows also..` in 
> For detailed information on a rule, the alerts it generated, and associated errors, click on a rule's name in the *All rules* table. This also allows you to perform the same actions...

refers more to what is possible after clicking on rule's name rather that what is possible to do on monitoring table.
Maybe, worth to highlight actions are possible on monitoring table?

Reply:
@vitaliidm I agree. We could reorder and rephrase slightly to lead with those actions:
```suggestion
You can perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules. For detailed information on a rule, the alerts it generated, and associated errors, click on a rule's name in the *All rules* table. 
```

Reply:
Thanks, @joepeeples. Looks good